src/agent/onefuzz-agent/src/local/common.rs

```diff
@@ -299,10 +299,11 @@ pub async fn wait_for_dir(path: impl AsRef<Path>) -> Result<()> {
         if path.as_ref().exists() {
             Ok(())
         } else {
-            Err(BackoffError::Transient(anyhow::anyhow!(
-                "path '{:?}' does not exist",
-                path.as_ref()
-            )))
+            let max_retry_period: Option<Duration> = Some(WAIT_FOR_MAX_WAIT);
+            Err(BackoffError::Transient {
+                err: anyhow::anyhow!("path '{:?}' does not exist", path.as_ref()),
```

Comment:
Values of type `&Path` do not impl the `Display` trait, but they provide a [`display()` method](https://doc.rust-lang.org/stable/std/path/struct.Path.html#method.display) which returns a thin wrapper type that does impl `Display`, so we can use `{}` instead of `{:?}`. This will result in my more readable error messages with (for example) less escaping of path separators.

Note that this is specific to `Path`/`PathBuf`.
```suggestion
                err: anyhow::anyhow!("path '{}' does not exist", path.as_ref().display()),
```

Reply:
Fixed!